iroha/Cargo.toml

```diff
@@ -41,6 +41,7 @@ iroha_schema = { path = "../iroha_schema"}
 
 async-trait = "0.1"
 chrono = "0.4"
+rcobs = "0.1"
```

Comment:
Do we need `rcobs` here? Can we do stream writing manually?

Reply:
We could. I will do that.